www/include/monitoring/recurrentDowntime/listDowntime.php

```diff
@@ -46,6 +46,9 @@
 if (isset($_POST['searchDT']) && $_POST['searchDT']) {
     $search = $_POST['searchDT'];
     $downtime->setSearch($search);
+} elseif (isset($_GET['search']) && $_GET['search'] && !isset($_POST['searchDT'])) {
```

Comment:
Why do you use search parameter ?
The good one is searchDT, so you can remove those lines.